build.gradle

```diff
@@ -14,13 +14,24 @@ java {
 }
 
 group 'net.neoforged.gradleutils'
-version '3.0.0-alpha.10'
+version '3.0.0-alpha.11'
 
 repositories {
-    mavenCentral()
+    mavenCentral {
+        content {
+            excludeGroup('com.diffplug.spotless')
+        }
+    }
     gradlePluginPortal()
 }
 
+// Spotless runs on J11
+configurations {
+    compileClasspath {
+        attributes.attribute Attribute.of('org.gradle.jvm.version', Integer), 11
+    }
+}
```

Comment:
Why are we manually manipulating the Java version attribute on the compile classpath? Could we not isolate the Spotless utils plugin into a separate sourceset?